portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,67 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+h1 {
+  text-align: center;
+}
+
+body {
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  text-align: justify;
+}
+
+.content-container {
+  max-width: 650px;
+}
+
+div {
+  width: 800px;
+  border: 25px solid;
+  border-color: rgba(50, 100, 230, 0.0);
+  padding: 50px;
+  margin: 0 50px;
+}
+
+
+table {
+  margin-left:auto;
+  margin-right:auto;
+}
+
+th {
+  text-align: left;
+}
+
+th, td {
```

Comment:
Separate selectors and declarations by new lines.

https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

Reply:
Done.